backend/models/fileRef.js

```diff
@@ -142,6 +142,24 @@ FileRef.getOriginalFile = function(account, model, fileName) {
 	return fetchFileStream(account, model, collection, fileName, true);
 };
 
+FileRef.getFile = (account, model, collName, ref_id) => {
```

Comment:
@sanmont3drepo  There's a bit of change since safetibase, so I think this branch needs a revisit upon merging latest of staging.

For e.g. fetchFile parameters changed on fetchFile, it takes account, model, colName

Reply:
I think it's a bit confusing to have a `getFile` and a `fetchFile`. If it's because this is an external facing function (to obscure the need to append `.ref`) then maybe we should call this `fetchFile` and rename the other one as `_fetchFile`